### DISCUSSION TITLE
I can't withdraw my mercatox doge [Solved]

### DISCUSSION THREAD

**Discussion by ManoNeutro** [dogecoin/dogecoin#3190] [General]
I've been trying for days to withdraw my dogecoins without success from Mercatox, detail, this problem comes from last year with the covid. I waited for them to resolve it until today, however, until today the problem has not been resolved. I suggested paying for them to solve it, since then, they have already sent me to the technical department and higher, but so far they have not solved the problem..  Sad I suggested that they send or exchange my dogecoins at market price, BTC 0.5 and pay for the losses I had sending money in the wallet, however they claim that the problem with doge will be solved soon, even though it's been more than a year since the problem.. Angry

I noticed contradictory things on the site, you can enter dogecoins on it, but you cannot withdraw or send to other users of the site, gets you stuck.. 

https://i.imgur.com/ogvUtGF.png
https://i.imgur.com/JYZksRr.png
https://i.imgur.com/spMOYyk.png
https://i.imgur.com/nGSYzcq.png
https://i.imgur.com/K7isvj8.png

Site: https://mercatox.com/

[solve] Mercatox allowed my withdrawal, here is my full review:

After reporting mercatox on reddit, github, twitter, facebook, google, police station, coinmarketcap, etc from the 6th to the 26th of December, my issue was resolved.

Note that I had to report it in all media to get a response at the height. They haven't fixed the doge transfer yet, it might even be true that they are fixing it.

I will not stop looking at people trapped in mercatox who were not the fight for what is theirs as I was, there are thousands of dollars stopped in mercatox with the deposit available.

The solution to the problem was strange. Hanna made an appointment with me so that on the 27th at 4 pm I would be online for her to make the transfer. At no time was the doge available for withdrawal, and yes it was done by them.

I also cannot help noticing that many messages I received from support were lies (as it seems to me), as there was no response from any department, not finance, authority or superior. Who appears there in the comment is a normal attendant. That is, does mercatox not have weak employees? I noticed this because Josh and Hanna were the ones who solved it, Murry seemed to try and the others didn't even care and sent automatic messages.

Reading what people say about mercatox, those who criticize say that they are taking advantage of people, keeping their money there without the right to withdraw. Those who defend her say she is very lazy.

Putting these two analyzes together, you deduce that you will only get what you want with this exchange if you react to its delay in some way, in chat, by email, or by reporting it. I also noticed that by email the solution was better than by chat.

My recommendation is not to put doge in mercatox until everything is resolved (they don't want to take the deposit). I thank the members of the bitcointalk forum who tried to help me in their own way, in the end I don't believe that your action really reached mercatox, but I thank you for your good intentions. Also, who shared and reported mercatox with me on twitter and reddit and github.

In the end I didn't get any response from a doge developer about what is happening with mercatox. I even tried, but it didn't work, what happened was that the doge community on twitter tried to find them and report them.

**Comment by GiverofMemory** [comment]
Maybe one of the devs can contact them, perhaps they are having an issue with a transaction not getting added to a block.

Another sad possibility is lots of exchanges are going insolvent so they may just be stringing you along and are stealing money from customers.  I hope not and hope for the best for you.

**Reply by ManoNeutro** [reply]
I managed to get nano, only dogecoin that I can't get and that's been a year ago.

**Reply by ManoNeutro** [reply]
Do you know if you have any dev for me to tag here and see if it can help?

**Reply by GiverofMemory** [reply]
All of the devs are here so they will see it eventually, and if they want to help they will.  One more piece of data may help, ask the exchange if they need any help from a developer to get doge working for them, and that you could let the devs know for them.

**Reply by ManoNeutro** [reply]
Have you seen them answering after having seen? It would be nice to have a sign of life. Can you send me the link?

**Reply by patricklodder** [reply]
Seen it, acted to get some more visibility on it, but I'm just a code monkey, not the ruler of the universe...

**Reply by ManoNeutro** [reply]
It would help a lot to know how your interaction with mercatox was trying to solve this problem, because I would disclose this.

**Reply by patricklodder** [reply]
I asked them to look at this ticket.

**Reply by ManoNeutro** [reply]
Have they told you anything so far?

**Reply by patricklodder** [reply]
A community member reached out to them by email and they said they are still working on it.

https://twitter.com/JustinScerini/status/1604242816367874048

**Reply by ManoNeutro** [reply]
They've been saying this for over a year now, I got into this in 2016, at the time there were several advertisements for them when asked about doge and nano: https://www.reddit.com/r/dogecoin/comments/ox3rl6/mercatox_withdrawal/

**Reply by ManoNeutro** [reply]
Do you have any contact with any doge developer? I would like to talk to him...

**Reply by patricklodder** [reply]
You are? 😂 What do you want me to do?

**Reply by ManoNeutro** [reply]
My problem was solved: https://www.reddit.com/r/dogecoindev/comments/zvgo9n/i_took_a_mercatox_scam_they_robbed_me_of_76000/

**Reply by patricklodder** [reply]
Glad it got resolved. What makes you say that no developer gave you any response?

**Reply by ManoNeutro** [reply]
I wanted a developer to contact them to find out if they respond to what they are solving and when it will be ready. Because the withdrawal is still blocked. Or if they don't give answers, I've had many people here who have helped me, on bitcointalk, twitter, reddit who have contacted mercatox and just had a standard response. Just to help with the analysis, so that people understand what they're getting into if they're going to mess with mercatox.

**Reply by patricklodder** [reply]
I think you did get a developer response. I apologize for disappointing you regarding the results from my efforts.

**Reply by ManoNeutro** [reply]
On the contrary, I thank, because for me it was with effort and denunciation from everyone that we had the answer that mercatox can make doge withdrawals.

**Reply by CHANDRAMA2** [reply]
> Maybe one of the devs can contact them, perhaps they are having an issue with a transaction not getting added to a block.
> 
> Another sad possibility is lots of exchanges are going insolvent so they may just be stringing you along and are stealing money from customers. I hope not and hope for the best for you.

Hello sir

**Reply by ManoNeutro** [reply]
> > Maybe one of the devs can contact them, perhaps they are having an issue with a transaction not getting added to a block.
> > Another sad possibility is lots of exchanges are going insolvent so they may just be stringing you along and are stealing money from customers. I hope not and hope for the best for you.
> 
> Hello sir

Hi, my problem has already been solved.
